### DISCUSSION TITLE
[Improvement] Allow negative values on bar charts

### DISCUSSION THREAD

**Discussion by frabul** [datalust/seq-tickets#1543] [Ideas]
I would like to use bar charts to get the performance of some performace statistics which could be negative too. 
The problem is that negative values are all reported as zero.
![immagine](https://user-images.githubusercontent.com/180771/162417289-b0078171-4370-4c7d-b1af-8d41155b8ccf.png)

**Comment by liammclennan** [comment]
I'm not able to reproduce negative values going to zero, however, I find a scaling problem with negative values.

![image](https://user-images.githubusercontent.com/271514/162539790-73c608e0-170f-4b2a-9c11-e21e6c9ac09b.png)

It may be related to what you are experiencing.

**Reply by nblumhardt** [reply]
Raised as a bug https://github.com/datalust/seq-tickets/issues/1545 👍
